test/unit/services/test_catalogue_item.py

```diff
@@ -1202,7 +1334,11 @@ def test_update_with_obsolete_replacement_catalogue_item_id(
             id=str(ObjectId()),
```

Comment:
Should ID be different here? Can `model_dump()` be used?

Reply:
No it should be the same, I think model dump can be used, but it would also need `is_obsolete` to be changed and ideally the `obsolete_replacement_catalogue_item_id` would also be None, so I don't think it would be worth it here as you would be dumping the expected output of the test rather than an initial model.